src/Components/Article.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import styled from 'styled-components';
+import Readme1 from '../images/max-van-den-oetelaar-buymYm3RQ3U-unsplash.jpg';
+import Readme2 from '../images/dylan-gillis-YJdCZba0TYE-unsplash.jpg';
+
+export const article = [
+  { header: 'Guide for beginners', paragraf: 'Join our growing community and let us support your well-being. If you are a beginner and do not dare to try yoga because you think you are too stiff and immobile for some of the positions, then you have come to the right place!' },
```

Comment:
Overall, the code seems well-written and organized. However, the article constant could have been named differently to avoid confusion with the Article component.